packages/desktop/components/drawers/network-config/drawers/components/NetworkCard.svelte

```diff
@@ -0,0 +1,47 @@
+<script lang="ts">
+    import { localize } from '@core/i18n'
+    import { NetworkConfigRoute, networkConfigRouter } from '@desktop/routers'
+    import { ClickableTile, Text, Icon, FontWeight, TextType } from '@ui'
+    import { Icon as IconEnum } from '@auxiliary/icon'
+    import { truncateString } from '@core/utils'
+    import { NetworkHealth } from '@core/network'
+    import NetworkStatusPill from './NetworkStatusPill.svelte'
+
+    export let name: string
+    export let address: string
+    export let status: NetworkHealth
+
+    function onTileClick(): void {
+        $networkConfigRouter.goTo(NetworkConfigRoute.ChainInformation)
+    }
+
+    function onQrCodeIconClick(): void {
+        $networkConfigRouter.goTo(NetworkConfigRoute.ChainDepositAddress)
+    }
+</script>
+
+<ClickableTile classes="bg-transparent border border-solid border-gray-200" onClick={onTileClick}>
+    <div class="w-full flex flex-col gap-5">
+        <div class="flex flex-row justify-between">
+            <div class="flex flex-row">
+                <Text type={TextType.h4} fontWeight={FontWeight.semibold}>
+                    {name}
+                </Text>
+            </div>
+            <NetworkStatusPill {status} />
+        </div>
+        <div class="flex flex-row justify-between">
+            <div class="flex flex-col">
+                <Text fontWeight={FontWeight.medium} color="gray-600">
+                    {localize('general.myAddress')}
+                </Text>
+                <Text type={TextType.h4} fontWeight={FontWeight.medium}>
```

Comment:
We need to use the monospace font here rather than the regular one (not sure which one it is :sweat_smile:)